code/modules/nano/interaction/default.dm

```diff
@@ -43,6 +43,8 @@
 		var/obj/machinery/Machine = src_object
 		if(Machine.emagged)
 			return STATUS_INTERACTIVE
+	if(istype(src_object, /obj/item/device/radio/borg/syndicate)) //Not ideal, but they can't access their own radio otherwise.
```

Comment:
Add CanUseTopic() to the /syndicate radio subchild in the radio.dm file, and then add the check for Syndicate borgs there and have it return 1.